CONTRIBUTING.rst

```diff
@@ -53,6 +53,8 @@ Workflow
 ********
 
 * This project follows the `gitflow`_ branching model. Please name your branch accordingly.
+   - Project maintainers first create a PR to merge a release branch into the ``develop`` branch, then they merge the release branch into the ``master`` branch locally and push to master.
```

Comment:
This one is clear to me, as long as this is meant as a definition about "what to do when making a release".  I don't have any opinion on the policy, but at least I understand what I'm supposed to do when I read it. Is the reason for adding this line to add emphasis that they should come through a PR to develop?  Otherwise it seems to just directly follow the gitflow model.

Reply:
> Is the reason for adding this line to add emphasis that they should come through a PR to develop? 

Yeah, exactly. Afaik, the gitflow model doesn't include platform-specific things like PRs, just basic git operations, so my aim here is to make it clear of what happens on the platform when a release branch is ready to merge into master for both maintainers and contributors.